app/views/proposal/_form.html.haml

```diff
@@ -40,3 +40,7 @@
                         - if can? :destroy, commercial
                           = link_to 'Delete', conference_program_proposal_commercial_path(@conference.short_title, @event.id, commercial.id),
                           :method => :delete, :data => { :confirm => 'Are you sure?' }, class: 'btn btn-danger'
+                    - if commercial.versions.last.whodunnit.to_i != @event.submitter.id
```

Comment:
I think we can show the name of the user that added the commercial even if that user is the submitter. So that our layout is consistent and explicit (always shows the same thing and we don't let users guess what it might mean when there is no name).